eosio/create_accounts.sh

```diff
@@ -18,74 +14,87 @@ store_env() {
 
 account_maker() {
     # echo creating account $1
-    echo -e "${GREEN}-----------------------creating account $1-----------------------${NC}"
+    echo -e "${GREEN}----creating account $1----${NC}"
     creation_response="$(curl -s "$faucet/create/$1")"
     if [ "$(from_json .success "$creation_response")" = false ]; then
         echo "failed creation: $creation_response" 1>&2
         exit 1
     fi
     # import keys and store them just in case something went wrong
     owner_private_key="$(from_json .data.account.owner.privateKey "$creation_response")"
-    store_env owner_private_key $owner_private_key
+    store_env $1_owner_private_key $owner_private_key
```

Comment:
I'm wondering if these variables might contain problematic characters, and should therefore be quoted (in all places, naturally).